src/manifest/pfm.rs

```diff
@@ -801,7 +856,7 @@ mod test {
         let (mut rsa, mut signer) = test_rsa();
 
         #[rustfmt::skip]
-        let pfm: owned::Pfm = from_str(r#"{
+        let pfm: owned::ParsedPfm = from_str(r#"{
```

Comment:
Oops, I hit approve before I remembered that this doesn't build. Was this supposed to be `crate::manifest::pfm::ParsedPfm`?

Reply:
Embarassing!

No idea how that slipped through.